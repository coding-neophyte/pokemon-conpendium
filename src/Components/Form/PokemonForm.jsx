import React from 'react'

export default function PokemonForm({ handleSubmit, handleSearch, pokemonName, }) {
    return (
        <div>
            <form  aria-label='form' onSubmit={handleSubmit}>
                <label>Pokemon Name</label>
                <input type='text' value={pokemonName} onChange={(e) => handleSearch(e.target.value)} />
                <button type='submit'>Search</button>
            </form>

        </div>
    )
}
