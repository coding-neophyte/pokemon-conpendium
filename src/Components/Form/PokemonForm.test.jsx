import { screen, render } from '@testing-library/react'
import PokemonForm from './PokemonForm'


it('should render pokemon form', () => {
    render(<PokemonForm />)
    const pokemonName = screen.getByText(/Pokemon Name/)

    expect(pokemonName).toBeInTheDocument()
    expect(pokemonName).toMatchSnapshot()
})
