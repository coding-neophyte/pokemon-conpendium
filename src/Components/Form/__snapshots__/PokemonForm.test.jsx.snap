// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render pokemon form 1`] = `
<label>
  Pokemon Name
</label>
`;
